.travis.yml

```diff
@@ -35,7 +35,9 @@ script:
   - npm run tslint
   # pr-check needs to run before test. test modifies package.json.
   - npm run pr-check
-  - npm run test
+  - npm run unitTests
+  # TODO: Merge tests into one group... Silly Gulp 4.0
```

Comment:
consider removing "silly" from the comment. 😄

Reply:
Really? I liked that comment :)